api/reference/custom_actions.adoc

```diff
@@ -0,0 +1,383 @@
+
+[[custom-actions]]
+== Custom Actions
+
+With button groups and custom buttons defined on model instances, the REST API provides visibility
```

Comment:
Might be better to say "records" here - "model instances" refers to implementation details